### ISSUE TITLE
Support for jest 27

### ISSUE THREAD

**Issue by ankrut** [unindented/custom-jquery-matchers#39]
In Jest 27 the helper `jest.addMatchers` has been removed. Instead it is recommended to use `expect.extend` which is not compatible with jest-jquery-matchers. As far as I understand `comperator` can be simply removed to get proper matchers.

**Comment by unindented** [comment]
I don't currently use this library, but I'm open to PRs. Feel free to submit @ankrut!

I feel it should be similar to https://github.com/unindented/custom-immutable-matchers/pull/44

**Comment by ankrut** [comment]
Yes, the PR would be very similar to [unindented/custom-immutable-matchers#44](https://github.com/unindented/custom-immutable-matchers/pull/44). Unfortunately, I am not able to run the project, i.e. `npm install` fails and so far no luck to fix it.

Therefore, I can provide a _blind_ PR, and maybe you can try your luck assuming you know best how run it.

However, my workaround is to extract the `compare` functions. For example in a `jest.setup.js` file.

``` ts
import * as matchers from 'jest-jquery-matchers'

const jqueryMatchers = {}

for (let key in matchers) {
    jqueryMatchers[key] = matchers[key]().compare
}

expect.extend(jqueryMatchers)
```

**Comment by unindented** [comment]
Up to you. You can submit a PR, and if it looks good and it passes all the check I'll merge it. Or you can do your workaround.
